rio_tiler/reader.py

```diff
@@ -247,13 +246,6 @@ def part(
             height=orig_vrt_height,
         )
 
-    if warp_vrt_option:
-        warnings.warn(
-            "warp_vrt_option will be removed in 2.0, use vrt_options",
-            DeprecationWarning,
-        )
-        vrt_options = vrt_options or warp_vrt_option
-
```

Comment:
I think you didn't remove the options in the function definition!

Reply:
oh yeah you're right